### DISCUSSION TITLE
Stop autoplaying thumbnail?

### DISCUSSION THREAD

**Discussion by pixiekat** [ReVanced/revanced-discussions#596] [Question] [answered]
Is there any way or setting to stop the thumbnail from autoplaying the video when browsing Youtube with ReVanced? I don't mean auto-play after a video ends; I mean when you're scrolling through the Home tab and you stop briefly and the video begins to play without sound. I know it's a feature of Youtube itself, but is there a patch to disable it?

**Comment by DraakjeYoblama** [comment, marked as answer]
You can disable it in YouTube settings under General>Playback in feeds

You can also mute those Autoplay thumbnails from the little mute button in the upper right corner of a playing thumbnail. It should remember that for all Autoplay thumbnails.

**Reply by pixiekat** [reply]
Thank you! I tried looking in autoplay and couldn't find anything! It was worded strangely so I didn't even think that was the correct setting.

**Comment by SciLor** [comment]


**Reply by HaCOn86** [reply]
no it not
